### ISSUE TITLE
Re-evaluate standard.talon

### ISSUE THREAD

**Issue by knausj85** [knausj85/knausj_talon#188]
- [ ] It seems to me that these commands should probably be terminal-specific?
```
cd: "cd "
grep: "grep "
elle less: "ls "
run L S: "ls\n"
run (S S H | S H): "ssh"
run vim: "vim "
run make: "make\n"
run make (durr | dear): "mkdir "
```

- [x] Should any of the below always be available, or perhaps moved to language-specific things?
```
args: 
	insert("()")
	key(left)
[inside] (index | array): 
	insert("[]") 
	key(left)
empty array: "[]"
list in it: 
	insert("[]") 
	key(left)
(dickt in it | inside bracket | in bracket): 
	insert("{}") 
	key(left)
block: 
	insert("{}") 
	key(left enter enter up tab)
```

- [x] These seems pretty C-specific and would probably be better off in a C-specific `code_function` list
```
word streak: 
	insert("streq()") 
	key(left)
word printf: "printf"
```
- [x] These seems pretty python-specific 
```
dunder in it: "__init__"
state past: "pass"
```

etc, etc.

**Comment by ym-han** [comment]
Yeah, I agree with most of this. 

The main thing I would like to be around all the time are args, empty array, and {}, since they are helpful also for prose and technical writing, though perhaps we could come up with less-programming-sounding names for those.

**Comment by fidgetingbits** [comment]
I've already switched a lot of this in my branch,  but wasn't really following  any strict rules so hadnt  thought much about a pr yet.

https://github.com/fidgetingbits/knausj_talon/blob/master/misc/standard.talon

 my solution for things like `empty array`  it's just to make all  types of parentheses/braces  able to be paired  together by saying the plural version or  optionally saying pair <type>.  I agree the current way is too  programming- specific to remember when you're not actually coding

 once the c  stuff from  my or manawars branch  is merged,  certainly the two C calls can get removed..

 I have a somewhat updated python lang file that I need to clean up a bit anyways,  so I can add the missing stuff  above from standard  if it's missing and send a pull request "soon" 

for the terminal stuff I have an actual https://github.com/fidgetingbits/knausj_talon/blob/master/apps/linux/commandline.talon file that I suspect we can use for most of it.  I think in general as long as its linux specific command line functions it doesn't need to be terminal-specific,  I right now the terminal-specific  stuff is more about the  short cuts for the  terminal emulator itself

**Comment by knausj85** [comment]
This one's really no longer relevant, closing.
